### ISSUE TITLE
Lexical error on line...

### ISSUE THREAD

**Issue by Martii** [kemitchell/spdx-is-osi.js#1]
Hello,

Trying out this package and I'm getting a rather unusual error message for myself in *node*@8.x.

``` json
"spdx-is-osi": "0.1.2"
```

``` js
var isOSIApproved = require('spdx-is-osi');
```
...
``` js
try {
  isOSIApproved('GPL-3.0');
} catch (err) {
  console.error(err);
}
```

Error thrown of:
``` sh-session
Function {
  message: 'Lexical error on line 1. Unrecognized text.\n...ause OR BSD-2-Clause-Patent OR BSD-3-Cla\n-----------------------^',
  hash: { text: '', token: null, line: 0 } }
```

Any suggestions please?

**Comment by Martii** [comment]
This error disappears with jslicense/spdx-is-osi.js#2

**Comment by Martii** [comment]
Closed by your uncredited cherry picking of #2 in 2cdd00aa1d9f0312abf60ace90a835d9ff0bb699...abd50af0820c750d54a54440d541cc9298665438
